### ISSUE TITLE
Checklist Mandatory doesn't seem to do anything?

### ISSUE THREAD

**Issue by o-o00o-o** [mohitbagra/vsts-extensions#37]
I expected Mandatory to prevent me from marking a Feature etc as done but it doesn't?

I create a new PBI or Feature, add a mandatory task (it has a *) set status done and it lets me.

What am I missing?

**Comment by jnloveday** [comment]
I am experiencing the same behavior.

**Comment by mohitbagra** [comment]
"Mandatory" doesnt block work item state to be Resolved. It doesnt create a work item rule, its just a way to suggest people that this checklist item needs to be completed before the work item is closed. It cannot block a work item save. Even if I somehow block work item state save from work item form UI, it still wont stop people from setting the state using other clients like rest api

**Comment by mohitbagra** [comment]
This is out of scope for the extension. So I am closing this issue.
